common/source/docs/common-airvolute-DroneCore-Suite.rst

```diff
@@ -0,0 +1,134 @@
+
+-  **Jetson Xavier NX**
+-  Cube flight controller
+-  Internal 5V power supply
+-  Power selector
+-  Peripheral connectors
+-  USB hub
+
+
+**DroneCore.Power** - bottom board with power electronic containing
+
+-  4 FOC UAVCAN ESCs - 20A permanent / 40A peak current
+-  Stacked configuration allows 8 ESCs
+-  Power sensor (battery voltage and current measurement)
+-  Analog inputs for battery cell monitoring
+-  SMBUS
+-  LED Driver for 4 x WS2812B strips
+
+
+
+
+DroneCore.Suite comes with both Cube and Xavier NX preconfigured and communicating together and able to control a quadcopter. Developers no longer need to solve basic communication and compatibility issues coming from connecting two different IT worlds. It They can start to concentrate on their application needs from the beginning.
+
+Accompanying documentation provides a fast learning curve to start using various peripherals to control the drone and create the desired application.
+Various software libraries (coming soon) further enrich the possibilities for the developer relieving him from complex configuration and programming tasks.
+
+DroneCore.Suite provides easy to use ROS based software stack supporting development of autonomous applications.
+
+Out of the box device is configured to fly a quadcopter (though flight controller calibration is needed anyway).
+
+---------------------------
+Top view
+---------------------------
+.. image:: ../../../images/airvolute_DCS_top_view_9219.jpg
+
+
+---------------------------
+Specifications:
+---------------------------
+
+**Mechanical parameters**
+
+-  Weight 247g - including Cube orange, Xavier NX, Xavier heatsink with
+   fan
+-  Dimensions 115 x 80 x 45mm
+-  Mount with 4 screws/spacers M3 109 x 74mm
+-  WiFi module
+-  WiFi antennas 2pcs
+
+**Electrical parameters**
+
+-  Power supply range: 12V – 35V (6S LiHV)
+-  Integrated DC/DC converter for control circuits
+-  Redundant power supply with power good monitoring for control unit
+-  Current protected peripheral connectors
+-  4 x FOC UAVCAN ESC 40A, featuring motor identification and motor
+   diagnostics
+-  Power sensor, SMBUS
+
+**Ardupilot (Cube) Connections**
+
+-  PPM input
+-  PWM output (7 + 7 lines)
+-  2x CAN
+-  4x UART
+-  2x I2C
+-  Buzzer
+-  Power sensor input
+-  ADC input
+-  Buzzer
+
+**Jetson connections:**
+
+-  PCI Express (M2, Key E connector)
+-  4 USB 3.0 (ZIF connectors, reductions available)
+-  Gigabit Ethernet (ZIF connector, reductions available)
+-  6 CSI (22 pin)
+-  4 GPIO
+-  UAV CAN, UART, I2C, SPI
+-  IMU BMI088 and barometer BMP388 on board
+-  USB 2.0 for debugging
+-  Micro SD card
+-  Video output connector (micro HDMI)
+-  Fan
+
+**DroneCore.Power** (bottom power board)
+
+-  4 x FOC UAVCAN ESCs – 40A peak / 20A permanent
+-  power sensor, SMBUS,
+-  LED Driver for 4x WS2812B strips
+
+
+---------------------------
+Block Schematic
+---------------------------
+.. image:: ../../../images/airvolute_DroneCore.Pilot_block_diagram.svg
+
```

Comment:
add  target link to show full view when double clicked:
```suggestion
```suggestion
    :target: ../../_images/airvolute_DroneCore.Pilot_block_diagram.svg
```
```